### DISCUSSION TITLE
Today's visitors on dashboard

### DISCUSSION THREAD

**Discussion by sancho-rod** [plausible/analytics#915] [Anything Else]
Show "today's visitors" instead or in addition to current "visitors in last 24h" which is not very meaningful/comparable.

**Comment by metmarkosaric** [comment]
thanks @sancho-rod! the reason we use last 24 hours is that it is the one that is the most comparable for many of our users. many people have several websites and they are in different timezones so today doesn't really work for those cases. last 24 hours works for all the cases even for those with many sites in different timezones

**Reply by sancho-rod** [reply]
thanks for your reply, marko! maybe it could be a choosable option one day for those who have clients/websites _only_ in their time zone / country ;)
